### ISSUE TITLE
Error with schema generator

### ISSUE THREAD

**Issue by Simperfit** [api-platform/schema-generator#126]
from  @vdebes at https://github.com/api-platform/api-platform/issues/653
app/config/schema.yml
```
  Person:
    properties:
      email: { unique: true, nullable: false }
      identifier: { range: "Text", unique: true, nullable: false }
```
src/AppBundle/Entity/Person.php
```
class Person
{
    /**
     * @var int|null
     *
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @var string email address
     *
     * @ORM\Column(type="text"unique=true)
     * @ApiProperty(iri="http://schema.org/email")
     * @Assert\Email
     * @Assert\NotNull
     */
    private $email;

    /**
     * @var string
     * @ORM\Column(type="text"unique=true)
     * @ApiProperty(iri="http://schema.org/identifier")
     * @Assert\NotNull
     */
    private $identifier;
```
The generated Person entity displays ```@ORM\Column(type="text"unique=true)``` which is incorrect and needs to be corrected before a schema update.

**Comment by Deuchnord** [comment]
Can't reproduce this on my side, can you @Simperfit?

**Comment by dunglas** [comment]
Closed byb 3cde07c2677c92162259ba976e0dad04aaee8691
